_posts/2015-12-16-API-Blueprint-Future.md

```diff
@@ -0,0 +1,100 @@
+---
+title: This is going to be Awesome
+excerpt: On the future of API Blueprint
+layout: post
+date: 2015-12-16 00:00:00 +0100
+author: zdenek
+published: true
+comments: true
+---
+
+What would API Blueprint be like if it started today? Different vocabulary, features or grammar concepts?
+
+The truth is, in the past three years API Blueprint was putting its flag in the sand. It was carving out its concepts and defining the emerging API design space.
+
+From its beginnings, API Blueprint stood out for its simplicity and collaborative nature. The aim has always been to create a simple, concise yet powerful language. We wanted to provide a medium for everybody to think about a design of an API.
+
+I am still shocked how much these core values resonated with the community. To ask programmers to write in plain text instead of JSON or XML was the bet we took. And the bet wasn't only about writing in freeform text. We also asked to not generate the API description from code. Instead we suggested it is better to take the effort and scope out an API definition before coding.
+
+After three years I can say our bet was well worth taking. API Blueprint is now growing faster then ever. It has way over 3,000 star gazers on [GitHub][1] and almost 200,000 blueprints in Apiary. It's now used everywhere from IoT devices to banks.
+
+### The most important feature
+
+So where is API Blueprint going next? Before looking into the future let's start with the present.
+
+In 2015, we have spent most of the year building only one feature—the description of body parameters. This feature is also know as the [MSON syntax][2].
+
+Only one feature? Why?
+
+Because this feature brings the single most important concept in understanding an API. It introduces the description of the application data model domain and its semantics.
```

Comment:
In reading this again, I think this clause is probably OK to leave. Even though it's a sentence fragment, it can probably work in this situation.